packages/app-admin-ui/client/components/Nav/index.js

```diff
@@ -289,8 +292,70 @@ function PrimaryNavItems({
   );
 }
 
-let PrimaryNavContent = ({ mouseIsOverNav }) => {
-  let {
+const UserInfoContainer = styled.div`
+  align-self: stretch;
+  padding: ${PRIMARY_NAV_GUTTER}px 0;
+  margin: 0 ${PRIMARY_NAV_GUTTER}px;
+  border-bottom: 2px solid ${colors.N10};
+  display: flex;
+  align-items: center;
+  font-size: 1.3em;
+`;
+
+const UserIcon = styled.div`
+  flex-shrink: 0;
+  width: 40px;
+  height: 40px;
+  border-radius: 50%;
+  background-color: ${colors.primary};
+  color: white;
+  display: flex;
+  flex-direction: column;
```

Comment:
```suggestion
```
With only one child item column is not doing anything here

Reply:
I needed it to get the `PersonIcon` vertically centered.

Reply:
Ah, wait, I see what you mean. Removed.